README.md

```diff
@@ -685,6 +685,15 @@ Specifies an option to specify whether this source should explicitly bypass any
 Requires at least Chocolatey v0.10.4.
 Defaults to `false`.
 
+#### `admin_only`
+
+(**Property**: This parameter represents a concrete state on the target system.)
+
+Option to specify whether this source should visible to administrators only.
```

Comment:
Administrators here being "Windows user accounts in the Administrators group", just so there is not confusion about what is meant here.

Reply:
Fixed!